snippets/snippets.json

```diff
@@ -0,0 +1,226 @@
+{
+    "bash if": {
```

Comment:
Two comments:

1. We have a 2-space indentation convention for every non-Go file. If accepted, this would have to be fixed before the merge.
2. I don't think I'd like to have snippets for bash commands like `if`/`else`/`sed`/`awk`/etc. I see that it would be more useful to have snippets for declaring tasks themselves. Maybe the basic Taskfile structure on a still empty file would be interesting as well.

Any opinions @pd93?

Reply:
Thanks for a feedback, I'll try to add snippets for tasks themselves.

Reply:
Hi, sorry for taking so long to look at this. It's been a very busy couple of weeks. I'm not sure that adding bash snippets really fits here. Especially given the conversation in https://github.com/go-task/task/issues/1115. We're generally trying to discourage users from adding large scripts directly into their Taskfiles for a [number of reasons](https://github.com/go-task/task/issues/1115#issuecomment-1501169579).

A PR that adds some snippets around tasks would be a useful addition though. Adding the `--init` boilerplate for example. Bear in mind that the YAML extension can already do some very simple things when it comes to suggestions, so its not work duplicating these.